CODEOWNERS

```
@@ -14,12 +14,13 @@
 
 @open-telemetry/erlang-approvers
 
-/instrumentation/opentelemetry_cowboy  @bryannaegele @tsloughter
-/instrumentation/opentelemetry_ecto    @bryannaegele @tsloughter
-/instrumentation/opentelemetry_nebulex @andrewhr
-/instrumentation/opentelemetry_oban    @indrekj
-/instrumentation/opentelemetry_phoenix @bryannaegele @tsloughter
-/instrumentation/opentelemetry_redix   @andrewhr
-/utilities/opentelemetry_telemetry     @bryannaegele @tsloughter
-/instrumentation/opentelemetry_tesla   @ricardoccpaiva
-/instrumentation/opentelemetry_httpoison @primait/shared-services
+/instrumentation/opentelemetry_cowboy           @bryannaegele @tsloughter
+/instrumentation/opentelemetry_ecto             @bryannaegele @tsloughter
+/instrumentation/opentelemetry_nebulex          @andrewhr
+/instrumentation/opentelemetry_oban             @indrekj
+/instrumentation/opentelemetry_phoenix          @bryannaegele @tsloughter
+/instrumentation/opentelemetry_redix            @andrewhr
+/utilities/opentelemetry_telemetry              @bryannaegele @tsloughter
+/utilities/opentelemetry_instrumentation_http   @tsloughter
+/instrumentation/opentelemetry_tesla            @ricardoccpaiva
+/instrumentation/opentelemetry_httpoison        @primait/shared-services
```

This file doesn't do much right now but just in case it can have non-org members as code owners, we can't have external teams as codeowners.
```suggestion
/utilities/opentelemetry_telemetry              @bryannaegele @tsloughter
/utilities/opentelemetry_instrumentation_http   @tsloughter
/instrumentation/opentelemetry_tesla            @ricardoccpaiva
```

---

I only added the `/utilities/opentelemetry_instrumentation_http` line, the other lines are changed only to keep the alignment (spaces added). Anyway you are right, github is marking red every user in this file beside you and @tsloughter.